test/json/__snapshots__/test.js.snap

```diff
@@ -1,5 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`delimiter - nest 1`] = `
+Object {
+  "a.hello": "hello",
+  "a.world": "world",
+  "b.hello": "hello",
+  "b.world": "world",
+}
+`;
```

Comment:
I pass it intentionally. If delimiter is "." , the result will be the same as "export json - nest" case. Because of "_" delimiter, result is flat object and not be separated into smaller object